util/arena.cc

```diff
@@ -8,13 +8,15 @@
 // found in the LICENSE file. See the AUTHORS file for names of contributors.
 
 #include "util/arena.h"
+#ifndef OS_WIN
 #include <sys/mman.h>
+#endif
+#include "port/port.h"
 #include <algorithm>
 #include "rocksdb/env.h"
 
 namespace rocksdb {
 
-const size_t Arena::kInlineSize;
```

Comment:
After removing it, it does not build under gcc anymore. Can you add it back if it is not Windows?